File: ejercicios/c++/ejercicio022.cpp
#include<iostream>

using namespace std;

int main() {

	int horas;
	int minutos;
	int segundos;

	cout<<"Ingrese la cantidad de horas (Formato 24 horas): ";
	cin>>horas;
	cout<<"Ingrese la cantidad de minutos: ";
	cin>>minutos;
	cout<<"Ingrese la cantidad de segundos: ";
	cin>>segundos;
	if (horas >= 0 && horas <= 23 && minutos >= 0 && minutos <= 59 && segundos >= 0 && segundos <= 59) {
		cout<<"Hora actual: "<<horas<<" : "<<minutos<<" : "<<segundos<<endl;
		segundos=segundos+1;
		if (segundos==60) {
			segundos=0;
			minutos=minutos+1;
			if (minutos==60) {
				minutos=0;
				horas=horas+1;
				if (horas==24) {
					horas=0;
				}
			}
		}
		cout<<"Hora despuŽs de un segundo: "<<horas<<" : "<<minutos<<" : "<<segundos<<endl;
	} else {
		cout<<"La Hora: "<<horas<<" : "<<minutos<<" : "<<segundos<<" es incorrecta."<<endl;
	}

	return 0;

}

File: ejercicios/python/ejercicio022.py
from math import *

horas = int(input("Ingrese la cantidad de horas (Formato 24 horas): "))
minutos = int(input("Ingrese la cantidad de minutos: "))
segundos = int(input("Ingrese la cantidad de segundos: "))

if horas >= 0 and horas <= 23 and minutos >= 0 and minutos <= 59 and segundos >= 0 and segundos <= 59:
    print("Hora actual: " + str(horas) + " : " + str(minutos) + " : " + str(segundos))
    segundos = segundos + 1
    if segundos == 60:
        segundos = 0;
        minutos = minutos + 1
        if minutos == 60:
            minutos = 0
            horas = horas + 1
            if horas == 24:
                horas = 0
    
    print("Hora despus de un segundo: " + str(horas) + " : " + str(minutos) + " : " + str(segundos))
else:
    print("La Hora: " + str(horas) + " : " + str(minutos) + " : " + str(segundos) + " es incorrecta.")  




